CHANGELOG.unreleased.md

```diff
@@ -10,6 +10,7 @@
 
 - [Home] Always sort the items by their names as a secondary sort criteria [#3983](https://github.com/vatesfr/xen-orchestra/issues/3983) (PR [#4047](https://github.com/vatesfr/xen-orchestra/pull/4047))
 - Properly redirect to sign in page instead of being stuck in a refresh loop
+- [Menu] XOA: Fix empty slot when no notifications nor updates [#4012](https://github.com/vatesfr/xen-orchestra/issues/4012) (PR [#4068](https://github.com/vatesfr/xen-orchestra/pull/4068))
```

Comment:
It doesn't indicate what the PR do. If the PR display an icon in this case, better mention it.